internal/exec/terraform_outputs.go

```diff
@@ -36,6 +36,7 @@ const (
 
 	varEnvVarPrefix    = "TF_VAR_"
 	cliArgEnvVarPrefix = "TF_CLI_ARGS_"
+	errorKey           = "error"
```

Comment:
No, add this instead to the .golangci-lint.yaml config as an acceptable string to repeat.

Reply:
okay cool